interfaces/builtin/system_source_code.go

```diff
@@ -0,0 +1,56 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+// https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#usrsrcSourceCode
+const systemSourceCodeSummary = `allows read-only access to /usr/src of the system`
```

Comment:
is this really meant to be "of" ? or "on" ? I would have expected either:

* access to the /usr/src of the system
* access to /usr/src on the system

Reply:
fwiw, I think it should be `on the system`.

Reply:
changed to `on the system`

Reply:
Sorry, yes, "on the system" is good. Thanks for updating it.